xray/handler.go

```diff
@@ -146,3 +148,54 @@ func btoi(b bool) int {
 	}
 	return 0
 }
+
+// traceHeaderID return trace id for http header
```

Comment:
Here we are actually generating value of key `_x_amzn_trace_id` which we are going to set in trace header. So, We can probably provide comment like `Generates value for _x_amzn_trace_id header key` or something similar to this. Also, can provide helper function name as `generateTraceIDHeaderValue` or something similar in this direction.